core/mono-samples/README.md

```diff
@@ -0,0 +1,7 @@
+# Sample .NET apps using the Mono runtime
+
+This folder contains sample code demonstrating how to build mobile (iOS, Android, Browser WebAssembly) apps with the mono runtime. The requirements for each sample can be found within the corresponding folder, and each sample can be ran using `dotnet publish`.
+
+## Prerequisites
+
+Dotnet sdk version 6.0.100-alpha.1.20531.2 is needed to run these samples. To install a specific version of the dotnet sdk, download the latest stable version of  `dotnet-install.sh` at <https://github.com/dotnet/install-scripts> and execute the script with `./dotnet-install.sh --version 6.0.100-alpha.1.20531.2`. Ensure that the version is correct with `dotnet --version` as the path may not be properly set.
```

Comment:
Let's make this a bit more generic. I use the powershell script for example. You'll have to merge this by hand as I didn't use full code blocks because the suggestion feature here won't work with it. Meaning, I used two backticks for the code blocks (bash and powershell) instead of three.

I also don't understand this clause:  Ensure that the version is correct with `dotnet --version` as the path may not be properly set.

```suggestion
.NET SDK version 6.0.100-alpha.1.20531.2 is needed to run these samples. To install a specific version of the dotnet sdk, download the latest stable version of  the dotnet-install script:

- Bash (Linux/macOS): <https://dot.net/v1/dotnet-install.sh>
- PowerShell (Windows): <https://dot.net/v1/dotnet-install.ps1>

Install version .NET version **6.0.100-alpha.1.20531.2**:

``bash
./dotnet-install.sh --version 6.0.100-alpha.1.20531.2
``
``powershell
./dotnet-install.ps1 -Version 6.0.100-alpha.1.20531.2
``
For more information, see [dotnet-install script reference](https://docs.microsoft.com/dotnet/core/tools/dotnet-install-script)
```

Reply:
Thanks! 
Ah, for more context, when installing dotnet sdk from one of the links in the chart at https://github.com/dotnet/installer, I believe the sdk will be installed at `/usr/local/share/dotnet/dotnet`. 
When I ran the `dotnet-install.sh` script to install `version 6.0.100-alpha.1.20531.2`, it was neither installed there nor at the default value of `/usr/share/dotnet` (`--InstallDir` under https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script)
I only realized this when checking the version of dotnet and seeing that it didn't match `6.0.100-alpha.1.20531.2` because my path pointed to `/usr/local/share/dotnet/dotnet`.
Instead, I saw the message: `dotnet-install: Adding to current process PATH: `/Users/mdhwang/.dotnet`. Note: This change will be visible only when sourcing script.` so it would be helpful for the user to check which dotnet will be used by checking the version and adding the correct path.
Note: I tried sourcing the script but it kept crashing the terminal window

Reply:
I suppose that the users can find out about `-InstallDir` from going to the docs as well